### PR TITLE
fix for issue #265

### PR DESCRIPTION
**Pull request by shakedel**
This commit is a simple solution for issue #265. However, using single letter group (e.g. f:/bar) will break. Shall I add a safety check?

### CODE REVIEW

**Comment by dandelionmane** [comment]
Could you please add some new tests in application_tests that verify the new behavior? Adding a safety check would be nice too; maybe if it thinks that it's detected a windows path, but that path doesn't exist, then it logs a warning and mentions that single letter run names are disallowed?

Also, please sign the CLA to satisfy googlebot.

**Comment by shakedel** [comment]
Cannot run tests on `master` since an *import* statement in line 32 of [tensorboard/tensorboard/plugins/profile/profile_plugin.py](https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/profile/profile_plugin.py) **fails**:
```from tensorboard.plugins.profile import trace_events_pb2```

**Comment by dandelionmane** [comment]
Hmm, that's interesting (/concerning). I'm not seeing that error on my end. Does `bazel run //tensorboard:tensorboard` work for you?

**Comment by shakedel** [comment]
A meaningless comment to make the googlebot run the CLA check again (after I was added to `ibm-contributors` google group)

**Comment by dandelionmane** [comment]
Hey @shakedel! Could you rebase the PR on master and get travis CI to pass, so we can resurrect this PR? It still would be really nice to get a fix for #52

**Comment by shakedel** [comment]
I added tests and merged `master` into the PR

File: tensorboard/backend/application_test.py [hunk 1]
```diff
@@ -186,55 +189,54 @@ class TensorboardServerUsingMetagraphOnlyTest(TensorboardServerTest):
 
 class ParseEventFilesSpecTest(tf.test.TestCase):
 
+  def customFsUtil(self, pathObj, logdir, expected):
```
Comment (dandelionmane) [inline tensorboard/backend/application_test.py]: `customFsUtil` is a strange name. This is effectively a custom assert, so maybe:
`assertPlatformSpecificLogdirParsing`
with a docstring that explains what pathObj is

**Comment by shakedel** [comment]
I signed it!

**Comment by shakedel** [comment]
God help me it was hard to satisfy the Google bot.

**Comment by shakedel** [comment]
@dandelionmane , I fixed all of the errors. You are welcome to review

**Comment by dandelionmane** [comment]
Thanks!

**Pull request merged**